### PR TITLE
Add task

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Use now can add task 

- Modal is updated with basic validation 
- Data is stored in the local storage to keep the task persistent

🔗  Preview: https://deploy-preview-10--clockwork-app.netlify.app/task

### CODE REVIEW

**Review by saadahmadhere — APPROVED** [review]
Astonishingly clear code. All good practices followed. Good to see the use of Regex.

File: src/components/Modal/Modal.jsx [hunk 1]
```diff
@@ -1,14 +1,25 @@
 import { useState } from "react";
 
 import { useEscape } from "../../hooks";
+import { updateTask } from "../task/utils/task-utils";
+import { addTask } from "./utils/add-task";
 import { HiddenLabel } from "./HiddenLabel";
```
Comment (MonikaJhingan) [inline src/components/Modal/Modal.jsx:7]: Can u please remove remove these extra spaces as that code quality is good but the in between spaces can make it more readable. Logic is well implemented.

Comment (junaidshaikh-js) [reply src/components/Modal/Modal.jsx:7]: The space is used to separate the blocks logically 

I separated the import from the dependencies and from src hence the space is given in between imports

Comment (MonikaJhingan) [reply src/components/Modal/Modal.jsx:7]: Ok that explains.

**Pull request merged**